tests/integration/test_logic_pipeline.py

```
@@ -0,0 +1,232 @@
+        return df, pd.Series(outcome, name='disease')
+
+    def test_complete_logic_flow(self, sample_medical_data):
+        """🔄 Test complete logistic regression pipeline from logic.py"""
+        df, y = sample_medical_data
+        
+        # Step 1: Validate input data
+        valid, msg = validate_logit_data(y, df)
+        assert valid is True, f"Data validation failed: {msg}"
+        
+        # Step 2: Run logistic regression
+        params, conf, pvals, status, metrics = run_binary_logit(y, df)
```

Ruff (0.14.10)

Unpacked variable `conf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)